InterfaceSkin object

shusaura85 opened the pull request:
Complete sprite replacement for INTERDEF.dat
Special thanks to @spacek531 for the trophy sprites

LeftofZen reviewed (changes requested):
The object properties are different to vanilla; can we leave them the same for now?

Yours:
![image](https://github.com/user-attachments/assets/1d6711e8-845b-42cb-995e-43604394e7ed)

Vanilla: 
![image](https://github.com/user-attachments/assets/04b74068-c389-4e23-a7c2-551530675c86)

LeftofZen commented on objects/InterfaceSkin/INTERDEF/0.png:
what was the reason to include 3 tabs horizontally here, instead of the 2 like vanilla?

shusaura85 replied:
Sprite 0 is used only as preview in object selection. Since the preview area is way bigger than the sprite (it can show sprites up to 110 x 110), I added a bigger sprite there.  
I initially tested with a sprite that was the size of the entire preview area (openloco icon), so when I went back to the toolbar preview, I added the extra tabs to be a bit more different from vanilla one

LeftofZen replied:
Fair enough, seems fine then

shusaura85 commented:
I updated the object to have the vanilla colors.  

> what was the reason to include 3 tabs horizontally here, instead of the 2 like vanilla?

Sprite 0 is used only as preview in object selection. Since the preview area is way bigger than the sprite (it can show sprites up to 110 x 110), I added a bigger sprite there.  
I initially tested with a sprite that was the size of the entire preview area (openloco icon), so when I went back to the toolbar preview, I added the extra tabs to be a bit more different from vanilla one

LeftofZen commented:
@shusaura85 you'll also need to remove the `.gitkeep` file. Those are just placeholders to stop git ignoring empty folders, so when files are added, they should be removed. I forgot about it in your other PRs but I fixed it in master, but we should make sure its removed here
